plugins/out_bigquery/bigquery.c

```diff
@@ -260,7 +260,7 @@ static flb_sds_t add_aws_signature(struct flb_http_client *c, struct flb_bigquer
 
     signature = flb_signv4_do(c, FLB_TRUE, FLB_TRUE, time(NULL),
                               ctx->aws_region, "sts",
-                              0, ctx->aws_provider);
+                              0, NULL, ctx->aws_provider);
```

Comment:
@matthewfala Do you know why we have AWS Auth in the bigquery output here?? Isn't that a google only product??

Reply:
Supposedly, AWS clients can use google cloud products with AWS credentials via [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation)

I see someone mention this usecase for fluent bit in the following comment: https://github.com/fluent/fluent-bit/issues/5084